Cargo.toml

```diff
@@ -203,6 +203,7 @@ xz2 = { version = "0.1.7" }
 zeroize = { version = "1.8.1" }
 zip = { version = "2.2.3", default-features = false, features = ["deflate", "zstd", "bzip2", "lzma", "xz"] }
 zstd = { version = "0.13.3" }
+diskus = { git = "https://github.com/sharkdp/diskus", version = "0.8.0" }
```

Comment:
We can't add Git dependencies, it'll break our crates.io publish

Reply:
I think we should just vendor any speed-ups because this is pulling in a bunch of new dependencies (e.g., a second version of Clap).